test/Chainweb/Test/Mempool.hs

```diff
@@ -1,9 +1,12 @@
--- | Tests and test infrastructure common to all mempool backends.
+{-# LANGUAGE TupleSections #-}
```

Comment:
Sort the `LANGUAGE` list (now here's my OCD talking)

Reply:
I'm surprised `stylish-haskell` didn't catch that :thinking: